src/shippingservice/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN GRPC_HEALTH_PROBE_VERSION=v0.4.7 && \
     chmod +x /bin/grpc_health_probe
 WORKDIR /app
 COPY --from=builder /app/target/release/shippingservice /shippingservice
-ENV PORT=50051
+ENV PORT=50052
 
-EXPOSE 50051
+EXPOSE 50052
```

Comment:
Should this line use ${SHIPPING_SERVICE_PORT} instead of 50052?

Reply:
Ya, I will do a followup PR that makes sure we do this for all services. It will also include only use the actual service port env var for example SHIPPING_SERVICE_PORT in this case rather than assigning a new env var named PORT which seems like a pointless mapping IMO.

Reply:
@mic-max I'm actually not able to test this PR due this.
The port 50052 is already allocated by FeatureFlagService and ShippingService is not able to start.

Reply:
Sorry, I fixed it now